pymmcore_plus/remote/_util.py

```diff
@@ -28,13 +28,32 @@ def _f(self, *args, **kwargs):
         _f.__name__ = name
         return _f
 
+    def _proxy_property(name):
+        def get(self):
+            print("here?")
+            return self._obj.mda
+
+        return property(get)
+        # return  property(lambda self: getattr(self._obj, name))
+
     _dict_ = {}
     for k, v in chain(*(c.__dict__.items() for c in reversed(cls.mro()))):
+        # if "mda" in k:
+        #     print(k, callable(v))
+        #     print(isinstance(v, property))
+        #     # print(k)
         if callable(v) and not k.startswith("_"):
             _dict_[k] = _proxy_method(k)
             for attr in dir(v):
                 if attr.startswith("_pyro"):
                     setattr(_dict_[k], attr, getattr(v, attr))
+        elif isinstance(v, property):
+            if "mda" in k:
+                print("here")
+            _dict_[k] = _proxy_property(k)
 
     _dict_["__init__"] = __init__
-    return type(f"{cls.__name__}Proxy", (), _dict_)
+    proxy = type(f"{cls.__name__}Proxy", (), _dict_)
+    print("mda" in _dict_)
```

Comment:
This returns true, which is why I'm confused about it not showing up on RemoteMMCore